apps/docs/typedoc.json

```diff
@@ -22,13 +22,18 @@
   "cacheBust": true,
   "hideGenerator": true,
   "plugin": ["typedoc-plugin-markdown", "typedoc-plugin-merge-modules"],
-  "filenameSeparator": "-",
   // ********************** //
   // Markdown Config
   // ********************** //
   "hideBreadcrumbs": true,
-  "hideInPageTOC": true,
-  "entryDocument": "index.md",
+  "membersWithOwnFile": ["Class", "Enum", "Interface"],
+  "typeDeclarationFormat": "table",
+  "parametersFormat": "table",
+  "interfacePropertiesFormat": "table",
+  "classPropertiesFormat": "table",
```

Comment:
@aditya172926 What was the rationale behind making this a table?

Reply:
I checked in the docs from master branch and it makes the constructor params, types etc. in a tabular format which has a better readability, instead of a list (default)

Reply:
The issue is that we reference the list in our docs, hence why this step is [failing](https://github.com/FuelLabs/fuels-ts/actions/runs/10558168024/job/29247108530?pr=2650#step:10:306).

I think for this PR - just keep it like for like, so back to the list. We can explore other means of displaying this information in future iterations.

Reply:
Alright.